### PR TITLE
Make openlabcmd -c option work

### PR DESCRIPTION
**Pull request by wangxiyuan**
openlab.conf works now. Use `openlab -c` to specified the config file.

### CODE REVIEW

**Review by kiwik — APPROVED** [review]
LGTM

File: openlabcmd/openlabcmd/cli.py [hunk 1]
```diff
@@ -91,9 +94,25 @@ def run(self):
             return 1
         self.args.func()
 
+    def _initConfig(self):
+        self.config = configparser.ConfigParser()
+        if self.args.config:
+            locations = [self.args.config]
+        else:
+            locations = ['/etc/openlab/openlab.conf',
+                         '~/openlab.conf'
```
Comment (Yikun) [inline openlabcmd/openlabcmd/cli.py]: seems miss a ','

**Review by Yikun — CHANGES_REQUESTED** [review]

**Comment by wangxiyuan** [comment]
recheck

**Review by liusheng — APPROVED** [review]
lgtm

**Comment by Yikun** [comment]
++, thanks for quick update.

**Pull request merged**